inf-clojure.el

```diff
@@ -703,7 +703,7 @@ If you are using REPL types, it will pickup the most approapriate
 (define-obsolete-variable-alias 'inf-clojure-arglist-command 'inf-clojure-arglists-form "2.0.0")
 
 (defcustom inf-clojure-arglists-form-lumo
-  "(pr-str (lumo.repl/get-arglists \"%s\"))"
+  "(do (set! lumo.repl/*pprint-results* false) (lumo.repl/get-arglists \"%s\"))"
```

Comment:
Let bind does not work in cljs bootstrap. Still I guess I could do it once. There is a problem though...it is global so I might actually need to rethink this solution.